go.work

```diff
@@ -3,4 +3,5 @@ go 1.18
 use (
 	.
 	./tests
+	parsers/toml
```

Comment:
I'm not sure if we should add this with this PR. Maybe a separate issue/PR for adding all of the package modules?

Otherwise, everything else seems ok :+1:

Reply:
Yes maybe, I'm not sure if I remember well but I believe tests failed without this.

Reply:
I've added a fix for the go.work and tests not running in the workflow. Can you rebase your PR on master?